CIP-governance-metadata-cc-rationale/README.md

```diff
@@ -0,0 +1,133 @@
+---
+CIP: ?
+Title: Governance metadata - Constitutional Committee vote rationale
+Category: Metadata
+Status: Proposed
+Authors:
+    - Ryan Williams <ryan.williams@intersectmbo.org>
+    - Eystein Magnus Hansen <eysteinsofus@gmail.com>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/?
+Created: 2024-07-17
+License: CC-BY-4.0
+---
+
+## Abstract
+
+The Conway ledger era ushers in on-chain governance for Cardano via [CIP-1694 | A First Step Towards On-Chain Decentralized Governance](https://github.com/cardano-foundation/CIPs/blob/master/CIP-1694/README.md), with the addition of many new on-chain governance artifacts.
+Some of these artifacts support the linking off-chain metadata, as a way to provide context.
+
+The [CIP-100 | Governance Metadata](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0100) standard provides a base framework for how all off-chain governance metadata should be formed and handled.
+But this is intentionally limited in scope, so that it can be expanded upon by more specific subsequent CIPs.
+
+This proposal aims to provide a specification for off-chain metadata vocabulary that can be used to give context to Constitutional Committee votes.
+
+## Motivation: why is this CIP necessary?
+
+### For GA submitters
```

Comment:
> ### For GA submitters

I would not use the abbreviation, but spell it out **Governance Action**

Reply:
Will do!
```suggestion
### For governance action authors
```